### PR TITLE
fix(Activity Tracker): Added support for event streams targets in Atracker v2 API

### PR DESCRIPTION
**Pull request by veenanr**
Signed-off-by: Veena Rao <veenar@us.ibm.com>

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: atrackerv2/atracker_v2_integration_test.go [hunk 1]
```diff
@@ -174,6 +174,31 @@ var _ = Describe(`AtrackerV2 Integration Tests`, func() {
...
+				EventstreamsEndpoint: eventstreamsEndpointPrototypeModel,
+			}
+
+			target, response, err := atrackerService.CreateTarget(createTargetOptions)
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(201))
+			Expect(target).ToNot(BeNil())
+
+			targetIDLink2 = *target.ID
```
Comment (padamstx) [inline atrackerv2/atracker_v2_integration_test.go:198]: Probably shouldn't share the link variable with the testcase above.
```suggestion
			targetIDLink3 = *target.ID
```

File: atrackerv2/atracker_v2_integration_test.go [hunk 2]
```diff
@@ -391,6 +416,28 @@ var _ = Describe(`AtrackerV2 Integration Tests`, func() {
...
+
+			eventstreamsEndpointPrototypeModel := &atrackerv2.EventstreamsEndpointPrototype{
+				TargetCRN: core.StringPtr("crn:v1:bluemix:public:messagehub:us-south:a/11111111111111111111111111111111:22222222-2222-2222-2222-222222222222::"),
+				Topic: core.StringPtr("my-test-topic"),
+				Brokers: []string{"kafka-x:9094"},
+				Password: core.StringPtr("xxxxxxxxxxxxx"),
+			}
+
+			replaceTargetOptions := &atrackerv2.ReplaceTargetOptions{
+				ID:             &targetIDLink2,
```
Comment (padamstx) [inline atrackerv2/atracker_v2_integration_test.go:429]:
```suggestion
				ID:             &targetIDLink3,
```

Comment (padamstx) [reply atrackerv2/atracker_v2_integration_test.go:429]: Note that the cleanup code should also include a section to delete the target whose id is targetIDLink3

**Review by padamstx — CHANGES_REQUESTED** [review]
I noticed a few changes needed in the integration tests to allow the tests to exercise all three target types independently.

**Review by padamstx — APPROVED** [review]
LGTM

Could you please post a screenshot of a successful run of the atrackerv2 integration tests and examples for atrackerv2?
(e.g. `cd atrackerv2`, `go test -tags=integration,examples`)?  Thanks!

Once we have those results, I'll go ahead and merge.

**Comment by veenanr** [comment]
image

**Comment by padamstx** [comment]
In the screen shot, it looks like you ran only unit and integration tests but not examples.  You can use `-tags=integration,examples` to run unit tests, int tests and examples all at once.

**Comment by veenanr** [comment]
[testRun.txt](https://github.com/IBM/platform-services-go-sdk/files/9577003/testRun.txt)

**Pull request merged**